scripts/run_docker

```diff
@@ -12,7 +12,7 @@ else
 fi
 export HOST
 docker-compose up -d
-docker cp app/server/src/resources/config.json beebop_beebop-server_1:/app/src/resources/config.json
+docker cp app/server/src/resources/config.json.in.docker beebop_beebop-server_1:/app/src/resources/config.json
```

Comment:
We were just using the wrong config file before in this script, which mean the redis location was wrong.